8.x/theme-development/basics.md

```diff
@@ -2,4 +2,190 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+In this tutorial, the basics of the Larammerce Template Engine will be explained. As you know, Larammerce is based on Laravel, and if you check a Larammerce template, you see that Larammerce blade files are the same as Laravel blade files. In the previous tutorial, a blade page was created, but in this tutorial, the general rules will be explained and then the template will be developed.
+
+.blade.php files are in the following directory `public/views/example.blade.php`.
+
+There are three types of blad files in this template engine.
+
+![blade-files](/blade-files-1-2.png)
+
+1- File name starts with underscore (partial files)
+
+2- File name does not start with underscore
+
+NOTE: A blade partial is similar to include or require in PHP. It’s an easy way to include contents of another file inside of a template.
+
+For example,  using the following commands extends the **_base.blade.php** page to other pages:
```

Comment:
These are not commands, these are some code pieces, So you can mention that:
Using the following piece of code tells the blade template engine to use the `_base.blade.php` file as the current page's layout.

Reply:
replace the `command` with `code` all over the document.